.run/main.run.xml

```diff
@@ -6,7 +6,7 @@
     <envs>
       <env name="PYTHONUNBUFFERED" value="1" />
     </envs>
-    <option name="SDK_HOME" value="docker-compose://[$PROJECT_DIR$/docker-compose.yml]:app_dev/python" />
+    <option name="SDK_HOME" value="docker-compose://[$PROJECT_DIR$/docker-compose-dev.yml]:app_dev/python" />
```

Comment:
Could you revert this change?

I mentioned this live as well, but we didn't have an easy way to test your changes end-to-end against realistic data. Something for me to think about as I'm going back to RecommendationAPI for the Setup Moment project.